content/en/docs/KRE/user/60_kais_runner_sdk/_index.md

```diff
@@ -97,6 +98,49 @@ func handler(ctx *kre.HandlerContext, data *any.Any) (proto.Message, error) {
 }
 ```
 
+## How to provide node functions to kre
+
+Once the `init` and `handler` functions have been declared and implemented, they have to be served in a certain way, so runners recognize them.
+
+### For runners V2
+
+#### Python V2
+
+Just by having two functions named __init__ and __handler__. Python runners interpret what source code is given.
+
+#### Golang V2
+
+Golang nodes use the Golang runners as a library. To load them up, inside the `main` function in the `main.go` file the runner must be run passing down the functions that serve as `init` and `handler`.
+
+```go
+func main() {
+  kre.Start(handlerInit, handler)
+}
+```
+
+### For runners V3
```

Comment:
This is confusing. So far we have v1 and v2 for Konstellation, v2 meaning the new branching workflow. What are v3 runners?

Reply:
The number of the version for the runners images we have uploaded to our registry.
I think I'll add further explanation, as python runners that support krt v2 are v3.0.0 and golang runners are the v4.0.0 images uploaded to the registry.